### PR TITLE
feat: add Helm Chart for new infrastructure

### PR DESCRIPTION
**Pull request by liviudm**

### CODE REVIEW

File: helm/fuel-explorer/values.yaml [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+
+image:
+  repository: ghcr.io/fuellabs/fuel-explorer
+  pullPolicy: Always
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "af1e196"
```
Comment (luizstacio) [inline helm/fuel-explorer/values.yaml:11]: Does this tag refer to some deployed images inside the repo?

File: .github/workflows/helm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
+name: Build and publish Helm Chart
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    types: [opened, synchronize]
+  release:
+    types: [published]
```
Comment (luizstacio) [inline .github/workflows/helm-publish.yml:10]: With this config, will the helm be published on every PR? should we make it just on the main branch? Or would this be capable of deploying preview envs?

**Review by luizstacio — CHANGES_REQUESTED** [review]

File: helm/fuel-explorer/values.yaml (hunk 1 above)
Comment (liviudm) [reply helm/fuel-explorer/values.yaml:11]: this is just a fallback. What gets deployed is set in the infrastructure repo https://github.com/FuelLabs/fuel-deployment-v2/pull/37/files - please see `kustomize/environments/fuel-dev-2/fuel-apps/dev-fuel-explorer/release.yaml` line 18

File: .github/workflows/helm-publish.yml (hunk 2 above)
Comment (liviudm) [reply .github/workflows/helm-publish.yml:10]: this is a good catch, thanks for the feedback. I just copied from your docker build pipeline. We should run this pipeline only if changes are made to `helm/**`. I'll update the rules

**Review by luizstacio — APPROVED** [review]

**Pull request merged**